# HG changeset patch
# User Kai NODA <nodakai@gmail.com>
# Date 1245127436 -7200
#      Tue Jun 16 06:43:56 2009 +0200
# Branch release-3-2-x
# Node ID 6c6e0d81148b051d5540157075652636757e991b
# Parent  1e640b3de2add5b00e5af8104b69887237ba6cc0
fix include guards

diff --git a/doc/interpreter/contributors.in b/doc/interpreter/contributors.in
--- a/doc/interpreter/contributors.in
+++ b/doc/interpreter/contributors.in
@@ -138,16 +138,17 @@ Stefan Monnier
 Antoine Moreau
 Kai P. Mueller
 Victor Munoz
 Carmen Navarrete
 Todd Neal
 Al Niessner
 Rick Niles
 Takuji Nishimura
+Kai Noda
 Eric Norum
 Krzesimir Nowak
 Michael O'Brien
 Peter O'Gorman
 Thorsten Ohl
 Arno Onken
 Luis F. Ortiz
 Scott Pakin
diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,13 @@
+2009-06-12  Kai NODA  <nodakai@gmail.com>
+
+	* ls-mat4.h: Fix include guard
+	* ov-type-conv.h: Add missing include guard.
+
 2009-06-12  Jaroslav Hajek  <highegg@gmail.com>
 
 	* ov-cell.cc (octave_cell::sort (Array<octave_idx_type>&)): Simplify.
 
 2009-06-10  John W. Eaton  <jwe@octave.org>
 
 	* pt-fcn-handle.cc (tree_anon_fcn_handle::dup): Don't convert to
 	tree_constant object here.  Do inherit from current symbol table
diff --git a/src/ls-mat4.h b/src/ls-mat4.h
--- a/src/ls-mat4.h
+++ b/src/ls-mat4.h
@@ -16,17 +16,17 @@ for more details.
 
 You should have received a copy of the GNU General Public License
 along with Octave; see the file COPYING.  If not, see
 <http://www.gnu.org/licenses/>.
 
 */
 
 #if !defined (octave_ls_mat4_h)
-#define octave_ls_mat4 1
+#define octave_ls_mat4_h 1
 
 extern oct_mach_info::float_format
 mopt_digit_to_float_format (int mach);
 
 extern int
 float_format_to_mopt_digit (oct_mach_info::float_format flt_fmt);
 
 extern int
diff --git a/src/ov-type-conv.h b/src/ov-type-conv.h
--- a/src/ov-type-conv.h
+++ b/src/ov-type-conv.h
@@ -15,16 +15,19 @@ FITNESS FOR A PARTICULAR PURPOSE.  See t
 for more details.
 
 You should have received a copy of the GNU General Public License
 along with Octave; see the file COPYING.  If not, see
 <http://www.gnu.org/licenses/>.
 
 */
 
+#if !defined (octave_ov_type_conv_h)
+#define octave_ov_type_conv_h 1
+
 static 
 octave_value
 octave_type_conv_body (const octave_value &arg, const std::string& name, int t_result)
 {
   int t_arg = arg.type_id ();
   octave_value retval;
 
   if (t_arg == t_result || arg.class_name () == name)
@@ -98,13 +101,15 @@ octave_type_conv_body (const octave_valu
     print_usage (); \
  \
   return retval
 
 #define OCTAVE_TYPE_CONV_BODY(NAME) \
   OCTAVE_TYPE_CONV_BODY3 (NAME, octave_ ## NAME ## _matrix, \
 			  octave_ ## NAME ## _scalar)
 
+#endif
+
 /*
 ;;; Local Variables: ***
 ;;; mode: C++ ***
 ;;; End: ***
 */
